src/identity.js

```diff
@@ -101,11 +103,16 @@ export class Identity extends EventEmitter {
         this.clientId = clientId;
         this.cache = new Cache(this.window && this.window.localStorage);
         this.redirectUri = redirectUri;
+        this.env = env;
         this.log = log;
 
         // Internal hack: set to false to always refresh from hassession
         this._enableSessionCaching = true;
 
+        // Internal hack: set to true if the SDK is being used inside the ITP iframe to
+        // avoid using the hasSession service and to prevent infinite iframe recursion
+        this._itpMode = false;
```

Comment:
I think I need to understand why we have this... Don't we control whether it's used in an iframe or not?